### DISCUSSION TITLE
Trending page

### DISCUSSION THREAD

**Discussion by filiptronicek** [open-sauced/open-sauced#663] [General]
I thought it would be cool to have a trending page (like on GitHub, could use the [GH Trending API](https://github.com/huchenme/github-trending-api)), could perhaps list the trending repos of the day, with a button to quickly add it to your goals. I know it perhaps wouldn't be found as useful as I want it to be, but I would use it, because of the ease of getting new repos to your future contributions.

A more advanced thing would be to able to set your favorite languages and get just the trending repos for that selection. That's something GitHub doesn't provide (yet), but could be achieved with the API.

**Comment by bdougie** [comment]
Hey @yg and I was chatting about this a few weeks ago. I would like to see something like this.

Are you able to spin a quick prototype using that API? Can you throw it on an HTML page or JavaScript app and link it here so we can see the data that is returned. 

Thanks for the idea.
